lib/manager/npm/post-update/index.ts

```diff
@@ -441,6 +441,10 @@ export async function getAdditionalFiles(
       './others/yarn',
       'YARN_CACHE_FOLDER'
     ),
+    YARN_GLOBAL_FOLDER: await ensureCacheDir(
+      './others/berry',
+      'YARN_GLOBAL_FOLDER'
+    ),
```

Comment:
```suggestion
    YARN_GLOBAL_FOLDER: await ensureCacheDir('./others/berry'),
```

I would prefer this, so we don't raise expectations which will break in v26.